packages/realm-react/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+x.x.x Release notes (yyyy-MM-dd)
+=============================================================
+### Enhancements
+* None.
+
+### Fixed
+* Adding event listeners while in a write transaction ([#4375](https://github.com/realm/realm-js/issues/4375))
```

Comment:
I think we should aim for changelogs which are more easy to understand for end users (e.g. what was the consequence of the bug, in addition to the underlying cause)

```suggestion
* Fixed potential "Cannot create asynchronous query while in a write transaction" error with `useObject` due to adding event listeners while in a write transaction ([#4375](https://github.com/realm/realm-js/issues/4375), since v0.1.0)

```